terraform/SLO_views_legacy_dataset.tf

```diff
@@ -0,0 +1,5 @@
+resource "google_bigquery_dataset" "SLO_views_legacy_dataset" {
+  dataset_id = "SLO_views_legacy"
+  project = "${var.slos_views_destination_project}"
+  location = "EU"
```

Comment:
location could also be exported as variable and configured default value

Reply:
Done